gfw_pixetl/utils/utils.py

```diff
@@ -142,7 +143,7 @@ def available_memory_per_process_mb() -> float:
 
 
 def get_co_workers() -> int:
-    return floor(GLOBALS.num_processes / GLOBALS.workers)
+    return max(GLOBALS.num_processes / GLOBALS.workers, 1)
```

Comment:
Is this an unrelated change? Would it be better in its own commit?

Reply:
Sorry, not sure how that got in there. It's a good change, but I'll stash it for a tiny addendum PR.